### PR TITLE
[Messenger] message consume command --queue parameter

### LINKED ISSUE #14919: [Messenger] Allow to limit consumer to specific queues
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#38973
| PR author(s) | @dbu
| Merged in    | 5.3-dev

### PR DESCRIPTION
**Pull request by dbu**
Feature introduced in https://github.com/symfony/symfony/pull/38973

fix #14919

I was considering whether to provide a full, meaningful example, but don't see where this would fit. Do you see a place where that should go? Otherwise i will write up the example as a blog post.

### CODE REVIEW

File: messenger.rst [hunk 1]
```diff
@@ -587,6 +587,29 @@ to handle messages in a priority order:
...
+cases to want a worker to only consume from specific queues.
+
+You can limit the worker to only process messages from specific queues:
+
+.. code-block:: terminal
+
+    $ php bin/console messenger:consume my_transport --queues=fasttrack
+
+At the time of writing, this option is only allowed with AMQP transports. To enabled
+it in your custom receiver, implement the QueueReceiverInterface.
```
Comment (Nyholm) [inline messenger.rst]: Maybe use a more "timeless" description. 

```
To enable use of the `queues` config, the receiver must `QueueReceiverInterface`.
```

**Review by Nyholm — COMMENTED** [review]
Thank you. 
I think it is good as it is. 

But yes, maybe a blog post will be helpful. @javiereguiluz will probably write something when it is time for "what's new in 5.3".

File: messenger.rst (hunk 1 above)
Comment (dbu) [reply messenger.rst]: i pushed an improvement

**Comment by dbu** [comment]
@javiereguiluz do you want me to write a guest post on the symfony blog?

**Review by Nyholm — APPROVED** [review]

File: messenger.rst [hunk 2]
```diff
@@ -587,6 +587,28 @@ to handle messages in a priority order:
...
+queues attached to the exchange of the specified transport. However, there are use
+cases to want a worker to only consume from specific queues.
+
+You can limit the worker to only process messages from specific queues:
+
+.. code-block:: terminal
+
+    $ php bin/console messenger:consume my_transport --queues=fasttrack
+
+To allow using the `queues` option, the receiver must implement the `QueueReceiverInterface`.
```
Comment (OskarStark) [inline messenger.rst]:
```suggestion
To allow using the ``queues`` option, the receiver must implement the :class:`Symfony\\...\\...\\QueueReceiverInterface`.
```

File: messenger.rst [hunk 3]
```diff
@@ -587,6 +587,28 @@ to handle messages in a priority order:
...
 there are none, *then* it will consume messages from ``async_priority_low``.
 
+.. _messenger-limit-queues:
+
+Limit Consuming to Specific Queues
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 5.3
+
+    Limiting the worker to specific queues was introduced in Symfony 5.3.
```
Comment (OskarStark) [inline messenger.rst]: Please move this down to the end of the paragraph, thanks

**Review by OskarStark — COMMENTED** [review]
Some minor comments

File: messenger.rst (hunk 2 above)
Comment (dbu) [reply messenger.rst]: oh, its been a while that i worked on .rst and not markdown :man_facepalming:

File: messenger.rst (hunk 3 above)
Comment (dbu) [reply messenger.rst]: done

File: messenger.rst [hunk 4]
```diff
@@ -587,6 +587,28 @@ to handle messages in a priority order:
...
+queues attached to the exchange of the specified transport. However, there are use
+cases to want a worker to only consume from specific queues.
+
+You can limit the worker to only process messages from specific queues:
+
+.. code-block:: terminal
+
+    $ php bin/console messenger:consume my_transport --queues=fasttrack
+
+To allow using the ``queues`` option, the receiver must implement the :class:`Symfony\\...\\...\\QueueReceiverInterface`.
```
Comment (OskarStark) [inline messenger.rst]: Can you please fill in the real namespace? 😄

Comment (dbu) [reply messenger.rst]: sorry. i thought you wanted to shorten it. but now i remember that it links to the class...

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thanks David for working on this feature, this is much appreciated.

**Pull request merged**